src/parser.ts

```diff
@@ -8302,6 +8337,8 @@ function parseClassElementList(
     key = parsePrivateIdentifier(parser, context | Context.InClass, tokenPos, linePos, colPos);
   } else if (context & Context.OptionsNext && (parser.token & Token.IsClassField) === Token.IsClassField) {
     kind |= PropertyKind.ClassField;
+  } else if (context & Context.OptionsNext && isStatic && token === Token.LeftBrace) {
```

Comment:
We don't need to check OptionsNext since this is a stage 4 feature.

Reply:
Oh yes, you're right. The time goes by so fast :-)